kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -301,6 +301,43 @@ struct GetCategories {
   }
 };
 
+struct GetLatestCategoryInfo {
+  const bool read_only = true;
+  std::string description() const {
+    return "getLatestCategoryInfo CATEGORY-ID [BLOCK-VERSION]\n"
```

Comment:
Nit: Maybe we can rename to `BLOCK-VERSION-FROM`, i.e. `...contains the given category, starting from BLOCK-VERSION-FROM`.

Reply:
Based on a comment below, maybe that should be called `BLOCK-VERSION-TO`. Did I get it correctly?

Reply:
done